### PR TITLE
[gcp]: Skip failing e2e tests

### PR DESCRIPTION
**Pull request by csrwng**
Skips currently failing e2e tests for gce provider.

### CODE REVIEW

**Comment by csrwng** [comment]
@abhinavdahiya needs an lgtm

**Comment by csrwng** [comment]
Removed skip for auth test. Fix delivered here: https://github.com/openshift/origin/pull/23577
Added comments to consolidate with: https://github.com/openshift/origin/pull/23582

**Comment by jsafrane** [comment]
/lgtm
from aos-storage point of view

File: test/extended/util/test.go [hunk 1]
```diff
@@ -476,6 +478,10 @@ var (
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Pre-provisioned PV \(ext4\)\] volumes should be mountable`,
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Pre-provisioned PV \(filesystem volmode\)] volumeMode should create sc, pod, pv, and pvc, read/write to the pv, and delete all created resources`,
 		},
+		"[Skipped:gce]": {
+			// Requires creation of a different compute instance in a different zone and is not compatible with volumeBindingMode of WaitForFirstConsumer which we use in 4.x
+			`\[sig-scheduling\] Multi-AZ Cluster Volumes \[sig-storage\] should only be allowed to provision PDs in zones where nodes exist \[Suite:openshift/conformance/parallel\] \[Suite:k8s\]`,
```
Comment (abhinavdahiya) [inline test/extended/util/test.go]: I think we need to drop the `[Suite:openshift/conformance/parallel\] \[Suite:k8s\]`

The test is not skipped in the run

File: test/extended/util/test.go [hunk 2]
```diff
@@ -476,6 +478,13 @@ var (
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Pre-provisioned PV \(ext4\)\] volumes should be mountable`,
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Pre-provisioned PV \(filesystem volmode\)] volumeMode should create sc, pod, pv, and pvc, read/write to the pv, and delete all created resources`,
 		},
+		"[Skipped:gce]": {
+			// Requires creation of a different compute instance in a different zone and is not compatible with volumeBindingMode of WaitForFirstConsumer which we use in 4.x
+			`\[sig-scheduling\] Multi-AZ Cluster Volumes \[sig-storage\] should only be allowed to provision PDs in zones where nodes exist`,
+
+			// Tries to ssh directly to a node. None of our nodes have external IPs
+			`[k8s.io] [sig-node] crictl should be able to run crictl on the node`,
```
Comment (abhinavdahiya) [inline test/extended/util/test.go]: we should add `\[sig-storage\] Flexvolumes should be mountable` as they require ssh too...

Comment (abhinavdahiya) [reply test/extended/util/test.go]: also this probably needs to be `\[k8s.io\] \[sig-node\] crictl should be able to run crictl on the node`

**Comment by abhinavdahiya** [comment]
I think we can skip `\[sig-auth\] Metadata Concealment should run a check-metadata-concealment job to completion`

https://github.com/kubernetes/kubernetes/blob/5a7b978c7401efad5672bcc876bcf5d3dfa71bd0/test/e2e/auth/metadata_concealment.go#L30

we already use openshift-sdn to conceal metadata https://github.com/openshift/sdn/blob/c9cbc6e2c8864a5421c8c63810610333179d4e52/cmd/sdn-cni-plugin/openshift-sdn_linux.go#L129-L133

**Comment by abhinavdahiya** [comment]
Also skip
`\[sig-storage\] Detaching volumes should not work when mount is in progress`

As it is also depends on SSH.

**Comment by abhinavdahiya** [comment]
based on https://github.com/kubernetes/kubernetes/issues/80767 esp https://github.com/kubernetes/kubernetes/pull/80769

it seems like we should open a bug for `\[sig-api-machinery\] AdmissionWebhook Should be able to deny pod and configmap creation` and skip it for now.

**Comment by abhinavdahiya** [comment]
https://github.com/openshift/release/pull/4735 should fix the `[sig-scheduling] Multi-AZ Cluster Volumes [sig-storage] should schedule pods in the same zones as statically provisioned PVs [Suite:openshift/conformance/parallel] [Suite:k8s]` error.

**Comment by wking** [comment]
/retest

@csrwng just ran `oc delete project ci-op-9ijgiymj` to clear any busted cache that might have lead to the broken `e2e-cmd` and `e2e-gcp` jobs.

**Pull request merged**